components/layout_2020/replaced.rs

```diff
@@ -489,6 +492,7 @@ impl ReplacedContents {
         block_sizes: &Sizes,
         inline_sizes: &Sizes,
```

Comment:
You could avoid making the clippy exception by making this argument `sizes: LogicalVec<&Sizes>`.

Reply:
Done. Added a helper function for this:

```rs
    pub(crate) fn as_ref(&self) -> LogicalVec2<&T> {
        LogicalVec2 {
            inline: &self.inline,
            block: &self.block,
        }
    }
```